### DISCUSSION TITLE
What are the basic values etc that are supported trying to decide which inverter to get.

### DISCUSSION THREAD

**Discussion by blair287** [nathanmarlor/foxess_modbus#679] [Q&A]
In tje table on the wiki that shows what Inverters support what I'm looking at maybe the KH7 but I want data for solar current power solar energy, battery soc , battery energy etc for my energy dashboard so I'm just looking for a more detailed table of what each inverter supports.

What am I loosing getting a KH on bus values and read wrote charge periods.

Thanks.

**Comment by canton7** [comment]
We don't (yet?) have a proper table of what entities each inverter supports (it would be nice to have, but we'd have to put some tooling in place to generate it).

It's not particularly user-friendly, but you can take a look through the files in https://github.com/nathanmarlor/foxess_modbus/tree/main/tests/__snapshots__/test_entity_descriptions.

All of the inverters should (unless I'm forgetting something major) support all of the basic stuff you need for the enregy dashboard, though.
